bundler/spec/update/gems/fund_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe "bundle update" do
+  let(:config) {}
+
+  before do
+    gemfile <<-G
+      source "#{file_uri_for(gem_repo1)}"
+      gem 'has_metadata'
+      gem 'has_funding', '< 2.0'
+    G
+
+    bundle :install
+  end
+
+  context "when listed gem is updated" do
```

Comment:
I see this is also coming from `spec/update/gems/post_install_spec.rb` but it doesn't make sense since `bundle update` is being run for all gems, not with a specific gem argument. So we could say

```suggestion
  context "when gems are updated" do
```

or something like that.